test/unit/messaging/messaging.spec.ts

```diff
@@ -2750,6 +2750,14 @@ describe('Messaging', () => {
           },
         },
       },
+      {
+        label: 'Generic fcmOptions message',
+        req: {
+          fcmOptions: {
+            analyticsLabel: 'test.analytics',
```

Comment:
I think these should be serialized as `analytics_label`. @chong-shao can you confirm if `analyticsLabel` is also acceptable?

Reply:
I tried "analyticsLabel" in the HTTP API. It is also acceptable.